src/_locales/es.config

```diff
@@ -61,40 +61,40 @@ Valor por defecto
 Fino
 
 @light
-Claro
+ligera
 
 @regular
 Regular
 
 @medium
-Medium
+Medio
 
 @bold
-Bold
+Negrita
 
 @black
-Black
+Negro
```

Comment:
should we call this Negra (similar to Italica negra)

Reply:
because spanish has gender, depends in what you are gonna use it with. if is gonna be used by itself it will be Negro.